flight/flight-sql-jdbc-driver/src/shade/NOTICE.txt

```diff
@@ -73,7 +73,7 @@ This product includes gRPC 1.70.0, with the following in its NOTICE:
 
 ---------------------------------------------------
 
-This product includes Netty 4.1.117.Final, with the following in its NOTICE:
+This product includes Netty 4.1.119.Final, with the following in its NOTICE:
```

Comment:
It seems that https://github.com/apache/arrow-java/tree/main/flight/flight-sql-jdbc-driver doesn't include copy of part of Netty. Do we need this? (Or should we change this to "This binary artifact contains ..."?)

Reply:
https://github.com/apache/arrow-java/blob/main/flight/flight-sql-jdbc-driver/pom.xml#L120

Reply:
I'm not familiar with Java but I thought `flight-sql-jdbc-driver-19.0.0-SNAPSHOT.jar` is a binary artifact not a source archive. (I thought that https://repository.apache.org/content/repositories/staging/org/apache/arrow/flight-sql-jdbc-driver/18.2.0/flight-sql-jdbc-driver-18.2.0-sources.jar is a source archive for flight-sql-jdbc-driver.)

We use "This binary artifact contains ..." in other files for binary artifact and we use "This product includes ..." for copied sources. e.g.: https://github.com/apache/arrow-java/blob/cf8b2521ca43c862784b67d32af94861dea1d388/LICENSE.txt#L205

So I thought:
* We don't need this for source archive?
* We should use "This binary artifact ..."?

BTW, I noticed that this isn't used for `-sources.jar`! This is used only for `.jar`, right?

Reply:
We have to ship LICENSE and NOTICE in the jar. That's why these files are in the shaded folder. And LICENSE has to list of dependencies shaded/shipped in the distributed artifact.